networks.json

```diff
@@ -1,316 +1,342 @@
+      },
+      {
+        "name": "Retired",
+        "chain_id": "cosmoshub-2",
+        "url": "https://cosmoshub-2.bigdipper.live"
+      },
+      {
+        "name": "Retired",
+        "chain_id": "cosmoshub-1",
+        "url": "https://cosmoshub-1.bigdipper.live"
+      }
+    ]
   },
   {
-      "name": "Akash",
-      "logo": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/logos/akash.svg?sanitize=true",
-      "cover": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/covers/akash.png?sanitize=true",
-      "links": [
-          {
-              "name": "Mainnet",
-              "chain_id": "akashnet-2",
-              "url": "https://akash.bigdipper.live"
-          },
-          {
-              "name": "Retired",
-              "chain_id": "akashnet-1",
-              "url": "https://akashnet-1.bigdipper.live"
-          }
-      ]
+    "name": "Akash",
+    "endpoint": "https://gql.akash.forbole.com/v1/graphql",
+    "links": [
+      {
+        "name": "Mainnet",
+        "chain_id": "akashnet-2",
+        "url": "https://akash.bigdipper.live"
+      },
+      {
+        "name": "Retired",
+        "chain_id": "akashnet-1",
+        "url": "https://akashnet-1.bigdipper.live"
+      }
+    ]
   },
   {
-      "name": "e-Money",
-      "logo": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/logos/e-money.svg?sanitize=true",
-      "cover": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/covers/e-money.png?sanitize=true",
-      "links": [
-          {
-              "name": "Mainnet",
-              "chain_id": "emoney-3",
-              "url": "https://emoney.bigdipper.live"
-          }
-      ]
+    "name": "e-Money",
+    "endpoint": "https://gql.emoney.forbole.com/v1/graphql",
+    "links": [
+      {
+        "name": "Mainnet",
+        "chain_id": "emoney-3",
+        "url": "https://emoney.bigdipper.live"
+      }
+    ]
   },
   {
-      "name": "Band Protocol",
-      "logo": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/logos/band.svg?sanitize=true",
-      "cover": "https://raw.githubusercontent.com/forbole/big-dipper-networks/main/covers/band-protocol.png?sanitize=true",
-      "links": [
-          {
-              "name": "Retired",
-              "chain_id": "band-guanyu-mainnet",
-              "url": "https://band-guanyu.bigdipper.live"
-          },
-          {
-              "name": "Mainnet",
-              "chain_id": "laozi-mainnet",
-              "url": "https://band.bigdipper.live"
-          }
-      ]
+    "name": "Band Protocol",
+    "endpoint": "https://gql.band.forbole.com/v1/graphql",
+    "links": [
+      {
+        "name": "Retired",
+        "chain_id": "band-guanyu-mainnet",
+        "url": "https://band-guanyu.bigdipper.live"
+      },
+      {
+        "name": "Mainnet",
+        "chain_id": "laozi-mainnet",
+        "url": "https://band.bigdipper.live"
+      }
+    ]
```

Comment:
In other places we firts put `Mainnet` and then `Retired`. Is there a specific reason why in this case it's inverted? Cause I think it would make more sense to have everything ordered the same unless there are specific reasons why in this case it's better to leave them this way

Reply:
I copied the config from https://github.com/forbole/big-dipper-networks/blob/main/networks.json#L64
I don't know the reason behind it, but I agree it will made more sense to change to ordering.

Reply:
done